config/build/crd/bases/build.projectriff.io_applications.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
+    controller-gen.kubebuilder.io/version: (devel)
```

Comment:
controller-gen v0.2.4 is required. If you delete the version on your path, running make will install the correct version.